ansible/deploy-clickhouse-proxy.yml

```diff
@@ -6,21 +6,18 @@
   become: true
   roles:
     - role: bootstrap
+    - role: nginx
+      tags: nginx
```

Comment:
I find it strange that nginx has to be explicitted here, given that `nginx` is defined as a dependency to the `dehydrated` role in here: https://github.com/ooni/devops/blob/main/ansible/roles/dehydrated/meta/main.yml